GruntFile.js

```diff
@@ -132,7 +132,8 @@ module.exports = function(grunt) {
                         'bower_components/d3/d3.js',
                         'src/*.js',
                         'src/models/*.js',
-                        'test/mocha/*.coffee'
+                        'test/mocha/*.coffee',
+	                    'https://cdn.rawgit.com/Kcnarf/d3-beeswarm/fbda9b54/build/d3-beeswarm.min.js',
```

Comment:
Ah... in that case, lets make the option throw an error (nv.log.error) about the option requiring that library and a link to the github project.  Basically, I'd like to not require it outright, but if it's there then the option will work.  We do the same thing with fastdom.